src/RSP_LoadMatrixNeon.cpp

```diff
@@ -0,0 +1,89 @@
+#include <algorithm>
+#include <cstring>
+#include "Debug.h"
+#include "RSP.h"
+#include "RDP.h"
+#include "N64.h"
+#include "F3D.h"
+#include "Turbo3D.h"
+#include "VI.h"
+#include "Combiner.h"
+#include "FrameBuffer.h"
+#include "DepthBuffer.h"
+#include "FrameBufferInfo.h"
+#include "GBI.h"
+#include "PluginAPI.h"
+#include "Config.h"
+#include "TextureFilterHandler.h"
+#include "DisplayWindow.h"
+#include "arm_neon.h"
+
```

Comment:
Why? These includes just useless there. Except for #include "arm_neon.h" in this case.
gSP.h don't need these includes too.